### PR TITLE
Administration pages

### PR DESCRIPTION
**Pull request by lowjoel**
Depends on #64, #65.

### CODE REVIEW

File: app/controllers/admin/controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Admin::Controller < ApplicationController
+  before_filter :authorize_admin
```
Comment (wangqiang1208) [inline app/controllers/admin/controller.rb]: @lowjoel  I think rails 4 prefers <code>before_action</code> over <code>before_filter</code>  ? 

https://github.com/rails/rails/commit/9d62e04838f01f5589fa50b0baa480d60c815e2c

**Comment by wangqiang1208** [comment]
So instance pages should be moved inside admin ?

**Comment by lowjoel** [comment]
@wangqiang1208 I think so... where else will you intend to put it?

**Comment by wangqiang1208** [comment]
well, fine.

**Comment by jsyeo** [comment]
There's a conflict here.

**Pull request merged**